include/psa/crypto_sizes.h

```diff
@@ -370,7 +370,18 @@
  *       just the largest size that may be generated by
  *       #psa_aead_generate_nonce().
  */
-#define PSA_AEAD_NONCE_MAX_SIZE 13
+#define PSA_AEAD_NONCE_DEFAULT_MAX_SIZE 13
```

Comment:
Please don't change this name. It's [defined by the PSA specification](https://armmbed.github.io/mbed-crypto/html/api/ops/aead.html?highlight=psa_aead_nonce_max_size#c.PSA_AEAD_NONCE_MAX_SIZE), albeit with a different specification which we now realize is incomplete. We arrived at the wording of the description in https://github.com/ARMmbed/mbedtls/pull/3787#discussion_r505482056. A future version of the PSA specification will adopt similar wording.